changelogs/fragments/display_verbosity.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - display itself should be single source of 'verboisty' level to the engine.
```

Should we deprecate `verbosity` in PlayContext in case 3rd party plugins are using it?

```suggestion
  - display itself should be single source of 'verbosity' level to the engine.
```